ariadne/cardano/test/backend/Test/Spec/Fixture.hs

```diff
@@ -46,26 +46,27 @@ genSpendingPassword :: PropertyM IO PassPhrase
 genSpendingPassword =
     pick (frequency [(20, pure emptyPassphrase), (80, arbitrary)])
 
-withLayerInMemoryStorage :: MonadIO m
-          => ProtocolMagic
-          -> (PassiveWalletLayer m -> Kernel.PassiveWallet -> IO a)
-          -> PropertyM IO a
+withLayerInMemoryStorage
+  :: (MonadIO m, MonadMask m)
+  => ProtocolMagic
+  -> (PassiveWalletLayer m -> Kernel.PassiveWallet -> IO a)
+  -> PropertyM IO a
 withLayerInMemoryStorage pm cc = liftIO $ withLayer pm Memory cc
 
 withLayerLocalStorage
-  :: MonadIO m
-  => ProtocolMagic
-  -> FilePath
-  -> (PassiveWalletLayer m -> Kernel.PassiveWallet -> IO a)
-  -> IO a
+    :: (MonadIO m, MonadMask m)
+    => ProtocolMagic
+    -> FilePath
+    -> (PassiveWalletLayer m -> Kernel.PassiveWallet -> IO a)
+    -> IO a
```

Comment:
Due to a decision by @manpages, we are moving to 2-space indentation. This is going to become mandatory after SRK-119 is resolved. Furthermore, indentation in this signature is inconsistent with indentation in `withLayerInMemoryStorage`. For now, please indent all signatures in this file with 2 spaces.